circuit_knitting/cutting/cutqc/dynamic_definition.py

```diff
@@ -46,10 +46,17 @@
 from typing import Sequence, Any
 from qiskit import QuantumCircuit
 from qiskit.circuit import Qubit
+from qiskit.utils.deprecation import deprecate_func
 
 from .wire_cutting_post_processing import build, find_process_jobs
 
 
+@deprecate_func(
+    removal_timeline="Circuit knitting toolbox 0.8.0 release",
+    since="0.7.0",
+    package_name="cutqc",
+    additional_msg="Use the wire cutting or automated cut finding functionality in the `circuit.knitting.cutting` module. ",
```

Comment:
```suggestion
    additional_msg="Use the wire cutting and automated cut-finding functionality in the `circuit_knitting.cutting` package. ",
```

Reply:
I looked into this and now agree with Caleb that it is a package, rather than a "module."

Reply:
:) https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/527#discussion_r1548102831